cfme/fixtures/authentication.py

```diff
@@ -13,6 +13,27 @@ def amazon_auth_provider():
         pytest.skip('amazon auth provider not found in auth_data.auth_providers, skipping test.')
 
 
+@pytest.fixture(scope='function')
+def ipa_auth_provider():
+    try:
+        return authutil.get_auth_crud('freeipa01')
```

Comment:
I don't think we should hardcode this, you should be able to parametrize `prov_key` based on what types of auth providers in the yaml, and use `auth_provider` fixture with the prov_key that matches the yaml.

freeipa01 is specific to our yaml config.

Reply:
I just used the same approach from previous fixture in this file.